pkg/dom0-ztools/rootfs/etc/sysctl.d/02-eve.conf

```diff
@@ -0,0 +1,59 @@
+# NOTE that EVE still has dynamic sysctl settings under
+# pkg/pillar/cmd/zedrouter/{ipsec,strongswan}.go
+#
+# Make sure to reconcile those when updating this file
+
+# global kernel settings
+kernel.softlockup_panic = 1
```

Comment:
This one is problematic on Hikey - hsysctl -w angs when setting it.
Does sysctl.d have a way to have conditional (on arch etc) files?